pkg/kotsutil/kots.go

```diff
@@ -272,6 +273,14 @@ func (o KotsKinds) Marshal(g string, v string, k string) (string, error) {
 					return "", errors.Wrap(err, "failed to encode identityconfig")
 				}
 				return string(b.Bytes()), nil
+			case "HelmChart":
+				if o.HelmCharts == nil {
+					return "", nil
+				}
+				var b bytes.Buffer
+				if err := s.Encode(o.HelmCharts, &b); err != nil {
+					return "", errors.Wrap(err, "failed to encode helmchart")
+				}
```

Comment:
Do we need a `return string(b.Bytes()), nil` here?

Reply:
I also called this case `"HelmChartList"` since that's the type that is being marshalled.

Reply:
definitely, great catch